third-party/CMakeLists.txt

```diff
@@ -175,7 +175,7 @@ set(LUA_COMPAT53_SHA256 bec3a23114a3d9b3218038309657f0f506ad10dfbc03bb54e91da7e5
 set(GPERF_URL https://github.com/neovim/deps/raw/ff5b4b18a87397a8564016071ae64f64bcd8c635/opt/gperf-3.1.tar.gz)
 set(GPERF_SHA256 588546b945bba4b70b6a3a616e80b4ab466e3f33024a352fc2198112cdbb3ae2)
 
-# 7za.exe cat.exe curl.exe ca-bundle.crt diff.exe tee.exe tidy.exe xxd.exe
+# cat.exe curl.exe ca-bundle.crt diff.exe tee.exe xxd.exe
```

Comment:
Should remove `cat.exe` instead?

Reply:
https://github.com/neovim/neovim/pull/10580

Reply:
I was confused by the title. Thanks for the explaining!